dependencies.yaml

```diff
@@ -323,3 +322,13 @@ dependencies:
         packages:
           # Already added to requirements via docs. This is for tests.
           - numpydoc>=1.5
+    specific:
+      - output_types: [conda, requirements, pyproject]
+        matrices:
+          - matrix: {cuda: "12.*"}
+            packages:
+              - cuda-python>=12.0,<13.0a0
+          - matrix: {cuda: "11.*"}
+            packages: &test_cuda_python_cu11
```

Comment:
out of curiosity, what does this `&test_cuda_python_cu11` notation do?

Reply:
This is a YAML anchor. We use this to define a list that we re-use in `*test_cuda_python_cu11`. We need a "fallback" list when the `cuda` selector is not defined.